apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts

```diff
@@ -17,7 +17,7 @@ export const Gstr1TransactionType = [
     { label: 'Credit Notes', value: 'credit-notes' },
     { label: 'Debit Notes', value: 'debit-notes' },
     { label: 'Advance Receipt', value: 'advance-receipt' },
-    { label: 'Adjusted advance receipt', value: 'adjusted-advance-receipt' },
+    // { label: 'Adjusted advance receipt', value: 'adjusted-advance-receipt' },
```

Comment:
will this commented code reuse in future?

Reply:
It's JV code commented for some time as new JV module is under development.